hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/utils/TestUtils.java

```diff
@@ -70,6 +70,16 @@ public static String getLastDeltaCompleteInstant(String basePath) {
         .orElse(null);
   }
 
+  public static String getLastCompletedCommitInstants(String basePath) {
+    final HoodieTableMetaClient metaClient = HoodieTableMetaClient.builder()
+            .setConf(HadoopConfigurations.getHadoopConf(new Configuration())).setBasePath(basePath).build();
+    return metaClient.getCommitsTimeline().filterCompletedInstants()
+            .filter(hoodieInstant -> hoodieInstant.getAction().equals(HoodieTimeline.COMMIT_ACTION))
+            .lastInstant()
```

Comment:
Is this resolved ?

Reply:
Sorry, I forgot to delete this when I was doing `cherry pick`.